Cargo.toml

```diff
@@ -46,3 +47,9 @@ uuid = { version = "1.2", features = ["v4", "serde", "js"] }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 uuid = { version = "1.2", features = ["v4", "serde"] }
+
+# For Development Purpose, patch the ssi* crates to use the local versions
+# [patch.crates-io]
+# ssi-claims = { path = "../ssi/crates/claims" }
+# ssi-dids = { path = "../ssi/crates/dids" }
+# ssi-jwk = { path = "../ssi/crates/jwk" }
```

Comment:
My preference is to keep this out of the Cargo.toml, as it makes assumptions about the developer's directory structure.